src/xrdok.js

```diff
@@ -681,3 +681,280 @@ const XRiconchevron = 'xr-icon-chevron';
+    for (var j = 0; j < Math.min(data.descriptors.length, data.primaryValues.length); j++) {
+      // get const
+      const prim = data.primaryValues[j]-offset;
+      const sec = (data.secondaryValuesBase * data.secondaryValues[j]/data.secondaryValuesBase)/100;
+      const linePath = document.createElement('a-entity');
+
+      // create primary vertices
+      planeGraphPrimaryGeo.vertices.push(
+        new THREE.Vector3(-data.secondaryValuesBase, prim, -j),
+        new THREE.Vector3(data.secondaryValuesBase, prim, -j)
+      );
+
+      // create secondary vertices
+      planeGraphSecondaryGeo.vertices.push(
+        new THREE.Vector3(-sec, prim + 0.001, -j),
+        new THREE.Vector3(sec, prim + 0.001, -j)
+      );
+
+      var step = j*2-2;
+
+      // create primary faces and material indices
+      if (j > 0) {
+        planeGraphPrimaryGeo.faces.push(
+          new THREE.Face3(step, step+1, step+2),
+          new THREE.Face3(step+1, step+3, step+2)
+        );
+        planeGraphPrimaryGeo.faces[step].materialIndex = step;
+        planeGraphPrimaryGeo.faces[step+1].materialIndex = step;
+      }
+
+      // create secondary faces and material indices
+      if (j > 0) {
+        planeGraphSecondaryGeo.faces.push(
+          new THREE.Face3(step, step+1, step+2),
+          new THREE.Face3(step+1, step+3, step+2)
+        );
+        planeGraphSecondaryGeo.faces[step].materialIndex = step;
+        planeGraphSecondaryGeo.faces[step+1].materialIndex = step;
+      }
+
+      // attach line path
+      planeGraphEl.appendChild(linePath);
+      linePath.setAttribute('line',
+        `start: ${-data.secondaryValuesBase} ${prim} ${-j};
+         end: ${data.secondaryValuesBase} ${prim} ${-j};
+         color: white;
+        `
+      );
+    }
+
+    // create primary material
+    planeGraphPrimaryMat.side = THREE.DoubleSide;
+    planeGraphPrimaryMat.transparent = true;
+    planeGraphPrimaryMat.opacity = 1;
+    planeGraphPrimaryMat.color.set( data.colorMin );
+
+    // create secondary material
+    planeGraphSecondaryMat.side = THREE.DoubleSide;
+    planeGraphSecondaryMat.transparent = true;
+    planeGraphSecondaryMat.opacity = 1;
+    planeGraphSecondaryMat.color.set( data.colorMax );
+
+    // create meshes
+    var planeGraphPrimaryMesh = new THREE.Mesh(planeGraphPrimaryGeo, planeGraphPrimaryMat);
+    planeGraphPrimaryEl.setObject3D('xr-primary-plane-graph', planeGraphPrimaryMesh);
+    var planeGraphSecondaryMesh = new THREE.Mesh(planeGraphSecondaryGeo, planeGraphSecondaryMat);
+    planeGraphSecondaryEl.setObject3D('xr-secondary-plane-graph', planeGraphSecondaryMesh);
+
+    planeGraphEl.appendChild(planeGraphPrimaryEl);
+    planeGraphEl.appendChild(planeGraphSecondaryEl);
+
+    planeGraphEl.setAttribute('visible', data.showGraph);
+    return planeGraphEl;
+  }
+
+  function createData(data, el) {
+    // parse CSV, if available
+    if (data.src) {
+      Papa.parse(data.src, {  // eslint-disable-line no-undef
```

Comment:
Please specify Papa in `globals` section of eslint.rc. then you can remove the comment line here

Reply:
I don't think we have an eslint.rc checked in yet, right?